### DISCUSSION TITLE
How to run on esp32 instead of esp32 s2 mini

### DISCUSSION THREAD

**Discussion by zx804262156** [simpleavr/s14clock#11] [Ideas]
How to run on esp32 instead of esp32 s2 mini

**Comment by simpleavr** [comment]
Not sure what you meant. If you mean to replace the S2 Mini w/ another ESP32 module, you will likely have to design a new PCB and working out the IO Pin to Segments / Digits mapping. You can find information from the source code, zf.pl is the perl script to create the maps. Start from there.
